documentation/src/main/asciidoc/topics/config_examples/application.properties

```diff
@@ -32,7 +29,7 @@ infinispan.remote.server-list=127.0.0.1:11222
 
 # Defines a comma-separated list of servers in this format:
 # `host1[:port],host2[:port]`.
-#infinispan.remote.serverList =
+#infinispan.remote.server-list=
```

Comment:
@danberindei Is there a list of the application properties somewhere or are they the same as Hot Rod client properties but with the `infinispan.remote` format?

Reply:
I don't think there is a list anywhere, unless @karesti knows about one.

Looking closer, what actually happens is that Spring Boot replaces kebab-cased property names with JavaBean-style setters and tries to invoke them on `InfinispanRemoteConfigurationProperties`, which extends the native client's `ConfigurationProperties` with additional properties like `client-properties` (a reference to another properties file in the native format). 

I haven't gone as far as to check it, but if `ConfigurationProperties` setter names match the names of the properties in `hotrod-client.properties`, they will also match the names of the properties in `application.properties` (except for the `infinispan.remote` prefix and the `-` instead of `_`).

There are also two "composite" properties, `infinispan.remote.cluster` and `infinispan.remote.sasl-properties`, I'm not sure how those work.

Reply:
thanks @danberindei I did a bit of digging but there doesn't seem to be a list. I've made some minor ish updates to this PR. hopefully it's good to go now. thanks for the feedback.

Reply:
@danberindei Hey, could you please take another look and merge if no other changes are required? Thanks!